File: quiz 2/quizQ8.js
//changing a given number into hours and minutes
function mathWizard(numberGiven) {
    var numberGiven = $("#numberGiven").val();
    var hours = Math.floor(numberGiven / 60);
    var minutes = numberGiven % 60;
    var resultText = numberGiven + " is " + hours + " hours and " + minutes + " minutes";
    return resultText;
}

$(document).ready(() => {
    $("#submitButton").on("click", () => {
        var numberGiven = $("#numberGiven").val();
        var resultText = mathWizard(numberGiven);
        document.write(resultText);
    });
});

//devide the numberGiven by 60 to get the hours then label the remainder as minutes

File: quiz 3/quizQ8.py
# changing a given number into hours and minutes


# devide the numberGiven by 60 to get the hours then label the remainder as minutes
numGiven =  int(raw_input("Enter a number to find out how many hours and minutes it is: "))
hours = numGiven / 60
minutes = numGiven % 60

def howLong():
    print "{} is {} hours and {} minutes".format(numGiven, hours, minutes)

howLong()